prompts/next-intl-add-language.prompt.md

```diff
@@ -0,0 +1,19 @@
+---
+mode: "agent"
+description: "Add new language to a NextJS with next-intl application"
+---
```

Comment:
It's likely you'll need some tools defined so that the agent can create/edit files.

Reply:
I have added tools. (using it today without)